.github/workflows/ci.yaml

```diff
@@ -48,15 +51,33 @@ jobs:
         name: codecov-umbrella # optional
         fail_ci_if_error: true # optional (default = false)
 
-    - name: Docker build
-      if: github.event_name == 'pull_request'
+    - name: Docker build # on PRs. on pushes to forks
+      if: github.event_name == 'pull_request' || (github.repository != 'keikoproj/upgrade-manager' && github.event_name == 'push')
```

Comment:
should we apply the condition `github.repository != 'keikoproj/upgrade-manager'` on both clauses?
when it's a pull request, does the github.repository equal to the fork the PR comes from?

Reply:
I wanted the CI run on PRs regardless of the repo. Personally I use PRs from dev-branches to master as part of my development process to be more organize and have cleaner commits. I thought that it could help other as well to check their changes against the CI before merging